### PR TITLE
Refs [Task] [44174] Build Gallery-section

### PR DESCRIPTION
**Pull request by tuanna-1205**
https://edu-redmine.sun-asterisk.vn/issues/44174

### CODE REVIEW

**Comment by tuanna-1205** [comment]
ready

File: app/views/blocks/gallery-section.pug [hunk 1]
```diff
@@ -0,0 +1,5 @@
+section.gallery 
+  .gallery__images
+    - var images = [{img: '../../images/gallery-1.jpg'}, {img: '../../images/gallery-1.jpg'}, {img: '../../images/gallery-1.jpg'}, {img: '../../images/gallery-1.jpg'}, {img: '../../images/gallery-1.jpg'}, {img: '../../images/gallery-1.jpg'}, {img: '../../images/gallery-1.jpg'}, {img: '../../images/gallery-1.jpg'}]
+      each src in images
+          img.gallery__images-img(src=src.img, alt="Gallery")
```
Comment (vietnq-1097) [inline app/views/blocks/gallery-section.pug]: Đưa images lên block vars nhé

```suggestion
block vars
    - var images = [{img: 'gallery-1.jpg'}, {img: 'gallery-1.jpg'}, {img: 'gallery-1.jpg'}, {img: 'gallery-1.jpg'}, {img: 'gallery-1.jpg'}, {img: 'gallery-1.jpg'}, {img: 'gallery-1.jpg'}, {img: 'gallery-1.jpg'}]
    - var imgSrc = '../../images/'

section.gallery 
  .gallery__images
      each src in images
          img.gallery__images-img(src=(imgSrc + src.img), alt="Gallery")
```

Comment (tuanna-1205) [reply app/views/blocks/gallery-section.pug]: vâng ạ

**Comment by tuanna-1205** [comment]
ready

**Pull request merged**